tests/unit/custom-granularity-calendar.spec.ts

```diff
@@ -368,48 +415,13 @@ describe('CustomGranularityCalendar', () => {
       // NOTE: to avoid being blocked in a sate when a user can't escape (with a start value out-of-bounds),
       // the navigation buttons are only visually disabled, but remains clickable.
 
-      await wrapper.setProps({
-        bounds: {
-          start: new Date('2021-05-01'), // start of may 2021
-          end: new Date('2021-12-01'), // start of december 2021
-        },
-      });
-
       expect(wrapper.find('.header-btn__previous').classes()).toContain(
         'custom-granularity-calendar__header-btn--disabled',
       );
       expect(wrapper.find('.header-btn__next').classes()).toContain(
         'custom-granularity-calendar__header-btn--disabled',
       );
 
-      // Disable the navigation towards the future, but not towards the past
-      await wrapper.setProps({
-        bounds: {
-          start: new Date('1990-01-01'), // start of 1990
-          end: new Date('2021-12-01'), // start of december 2021
-        },
-      });
-      expect(wrapper.find('.header-btn__previous').classes()).not.toContain(
-        'custom-granularity-calendar__header-btn--disabled',
-      );
-      expect(wrapper.find('.header-btn__next').classes()).toContain(
-        'custom-granularity-calendar__header-btn--disabled',
-      );
-
-      // Disable the navigation towards the past, but not towards the future
-      await wrapper.setProps({
-        bounds: {
-          start: new Date('2021-05-01'), // start of may 2021
-          end: new Date('2049-01-01'), // start of 2049
-        },
-      });
-      expect(wrapper.find('.header-btn__previous').classes()).toContain(
-        'custom-granularity-calendar__header-btn--disabled',
-      );
-      expect(wrapper.find('.header-btn__next').classes()).not.toContain(
-        'custom-granularity-calendar__header-btn--disabled',
-      );
```

Comment:
Why are we deleting all those assertions ? They are unnecessary ?

Reply:
Nop there are just not relevant since we move nav to bound start (it was not moving before)